packages/node_modules/@node-red/editor-client/locales/pt-BR/editor.json

```diff
@@ -0,0 +1,1125 @@
+                "repositoryUpToDate": "Seu repositório está atualizado.",
+                "commitsAhead": "Seu repositório é __count__ commit antes do remoto. Você pode enviar este commit agora.",
+                "commitsAhead_plural": "Seu repositório é __count__ commits antes do remoto. Você pode enviar esses commits agora.",
+                "commitsBehind": "Seu repositório é __count__ commit atrás do remoto. Você pode puxar este commit agora.",
+                "commitsBehind_plural": "Seu repositório é __count__ commits atrás do remoto. Você pode puxar esses commits agora.",
+                "commitsAheadAndBehind1": "Seu repositório é __count__ commit behind e",
+                "commitsAheadAndBehind1_plural": "Seu repositório é __count__ commits atrás de e",
+                "commitsAheadAndBehind2": "__count__ confirmar antes do remoto.",
+                "commitsAheadAndBehind2_plural": "__count__ confirma antes do remoto.",
+                "commitsAheadAndBehind3": "Você deve baixar o commit remoto antes de enviar.",
+                "commitsAheadAndBehind3_plural": "Você deve baixar os commits remotos antes de enviar.",
+                "refreshCommitHistory": "Atualizar histórico de commits"
+            }
+        }
+    },
+    "typedInput": {
+        "type": {
+            "str": "string",
+            "num": "número",
+            "re": "expressão regular",
+            "bool": "boolean",
+            "json": "JSON",
+            "bin": "buffer",
+            "date": "timestamp",
+            "jsonata": "expression",
+            "env": "env variable",
+            "cred": "credential"
+        }
+    },
+    "editableList": {
+        "add": "adicionar",
+        "addTitle": "adicionar um item"
+    },
+    "search": {
+        "empty": "Nenhuma equivalência encontrada",
+        "addNode": "adicionar um nó..."
+    },
+    "expressionEditor": {
+        "functions": "Funções",
+        "functionReference": "Referência de função",
+        "insert": "Inserir",
+        "title": "JSONata Expression editor",
+        "test": "Teste",
+        "data": "Mensagem de exemplo",
+        "result": "Resultado",
+        "format": "formato da expressão",
+        "compatMode": "Modo de compatibilidade habilitado",
+        "compatModeDesc": "<h3> Modo de compatibilidade JSONata </h3> <p> A expressão atual parece ainda fazer referência a <code> msg </code>, então será avaliada no modo de compatibilidade. Atualize a expressão para não usar <code > msg </code>, pois este modo será removido no futuro. </p> <p> Quando o suporte JSONata foi adicionado pela primeira vez ao Node-RED, era necessária a expressão para fazer referência ao objeto <code> msg </code> . Por exemplo, <code> msg.payload </code> seria usado para acessar a carga. </p> <p> Isso não é mais necessário, pois a expressão será avaliada em relação à mensagem diretamente. Para acessar a carga, o expressão deve ser apenas <code> carga útil </code>. </p> ",
```

Comment:
Change to:
`<h3> Modo de compatibilidade JSONata </h3> <p> A expressão atual parece ainda fazer referência a <code> msg </code>, então será avaliada no modo de compatibilidade. Atualize a expressão para não usar <code > msg </code>, pois este modo será removido no futuro. </p> <p> Quando o suporte JSONata foi adicionado pela primeira vez ao Node-RED, era necessária a expressão para fazer referência ao objeto <code> msg </code> . Por exemplo, <code> msg.payload </code> seria usado para acessar o conteúdo. </p> <p> Isso não é mais necessário, pois a expressão será avaliada em relação à mensagem diretamente. Para acessar a o conteúdo, a expressão deve ser apenas <code>payload</code>. </p>`